project.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<p>why are some of your charts missing axis labels?</p><p> </p><p>use more verbose variable name than. What does <code>reg</code> mean? Regular?</p><p> </p><p>Are your features on the same order of magnitude? Might you be able to get away without normalising for certain models? Why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-42FraMeWork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>